### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by mnapoleon** [argoproj-labs/argocd-image-updater#855] [Q&A] [answered]
Is there a schedule for the next release?  We are looking forward to using it with this fix merged. https://github.com/argoproj-labs/argocd-image-updater/pull/790

**Comment by jannfis** [comment, marked as answer]
No schedule yet, but I guess it's about time, >40 commits since v0.14.0. Guess we could roll one up very soon. Stay tuned :)

**Reply by mnapoleon** [reply]
Thanks for the update. I will keep a look out for it.

**Comment by ceguimaraes** [comment]
Hi guys. Any updates on this?
My team has moved to a multi source strategy on argocd and I am looking to get this fix: https://github.com/argoproj-labs/argocd-image-updater/pull/790

**Comment by chengfang** [comment]
the team is working on it, should be out very soon.

**Comment by jannfis** [comment]
0.15.0 has been released earlier this week.
